### DISCUSSION TITLE
Création d'une nouvelle classe

### DISCUSSION THREAD

**Discussion by Electek** [Crownicles/Crownicles#2491] [Nouvelles idées]
J'aimerais bien proposer l'addition d'une nouvelle classe: le Monk 🙏/ prêtre kyutiste

Raison: le monde de draftbot est centrée sur de la religion. L'église kyutiste joue un grand rôle dans le monde et pourtant nous ne pouvons pas totalement incarné un croyant.

Fonctionnement : j'avais l'idée que le monk utiliserait les attaques de l'adversaire contre lui. Les attaques que j'ai faites sont pas ouf donc si qq a des idées a proposé.

Exemple d'attaque que j'ai imaginé :

Contre attaque : inflige 70 a 150 pourcent de l'attaque de l'adversaire.
🌬️3

Bouddha : diminue l'attaque de l'adversaire de 90 pourcent mais diminue votre défense de 30 pourcent 
🌬️6

Bénédiction 

Lit (je ne sais plus comment elle s'appelle cet attaque X) )

Coût du lapin : paralysée votre adversaire (fait autant de dégâts qu'une simple)
🌬️4

Stats: pourquoi pas faire des stats tant que j'y suis x) . Bon là vrm je suis sur que sa va pas être comme sa j'ai abusé x)

Stats:
🛡️ 249
🗡️ 264
🚀7
🌬️5
🫁4

Bon je sais que niveau stats sa va être dur de faire sa sans ressembler a une classe déjà existante. En tout cas sa serais chouette de pouvoir jouer un kyutiste sur db

**Comment by aureochocob0n** [comment]
L'attaque bouddha a l'air un peux cheatée, sinon je trouve que l'idée est vraiment bien. Peut-être inverser le coût en souffle entre coup du lapin et contre attaque. Pourquoi pas ajouter aux attaques des effets passifs en fonction de L'attaque qu'à fait l'adversaire avant

**Comment by Gabrielre14** [comment]
Ce qui pourrait être intéressant serait d'ajouter une attaque qui met le joueur en altération positive. Par exemple, à la place du repos, une attaque qui te mets en altération "soin", permettant de régénérer autant qu'une poison (70 par tour) sur environ 3 tours.

**Reply by Electek** [reply]
oh pas mal sa

**Reply by LaMangousteRose** [reply]
J'aurais dit un peu moins

**Comment by kirit000** [comment]
en vrai c'est tes idées proposées elec sinon jpense que cv pas être comme ça xd, prck la contre attaque a le même nombre de souffles que la rapide alors que c'est incomparable en terme de puissance, pk pas lui rajouter du souffle en vrai, sinon j adore ton idée je vote pour.

**Comment by StrangerIM** [comment]
J'aime beaucoup !

**Comment by LaMangousteRose** [comment]
J'aime bien mais ce serait moins cheat sans lit et avec plutôt une attaque sur plusieurs tours comme chargée mais un peu nerf.

**Comment by Carsebel** [comment]
Ok pour de nouveau dans les classes mais pas que une seule mais un palier complet. De quoi laisser le choix aux joueurs

**Reply by LaMangousteRose** [reply]
Je suis complètement d'accord

**Reply by LaMangousteRose** [reply]
Pour tier 1 je propose Deacon (diacre)

**Reply by Carsebel** [reply]
Pas en lien avec le kyutisme serieux. 
Y a mieux à créer que se baser sur une ''religion''

**Reply by LaMangousteRose** [reply]
Ah bien vu

**Comment by aureochocob0n** [comment]
Ouais, pourquoi pas mettre ça en T6 à partir du lv100. Avec d'autres classes qu'on créerait à l'occasion

**Reply by StrangerIM** [reply]
Faudrait la faire légèrement plus puissante si c'est une classe T6, mais en faisant en sorte que ce soit la moins puissante ( en général un moine c'est pas fait pour se battre ) genre un peu comme les tanks 
Et si c'est le cas faudrait lui donner des avantages pour l'aventure comme par exemple une vie + élevée ( comme les tanks )

**Reply by Electek** [reply]
je pensait la mettre en t4 et 5 perso (et pq pas t6 dc) mais pour repondre a stranger il ya bcp de moine combattant dans le monde. comment l'eglise kyutiste protege t'elle ses lieux sacre? je pensait que sa proviendrais des dieux qui leur accorde un bonus et dc un bonus justement en combat. Apres, sa c'est ma vision ^^

**Reply by Carsebel** [reply]
On va éviter de mettre des classes divines car déjà que ça se sent plus quand certains se donnent le rôle d'empereur ou impératrice, alors si on mets une classe divine, ils vont choper le melon

**Reply by aureochocob0n** [reply]
T'es premier degré là?

**Reply by Cirpacha** [reply]
En vrai oui, l'idée de nouvelles classes est bonne, mais pas forcément une axée sur une pseudo religion pitié

**Reply by Jajadu69** [reply]
Je comprends pas trop le rapport qu'à ta réponse avec la suggestion . Entre une  classe axée sur la divination et le fait que certains se donnent le rôle d'empereur/ impératrice

**Reply by Cyrpa** [reply]
Car certains se permettent d'être malpolis et irrespectueux envers les autres parce qu'ils s'autoproclament "empereurs" où je ne sais quoi, et que personne n'a envie de subir la mauvaise foi de ces joueurs en version amplifiée à cause d'une simple classe. (C'est certes un autre débat qui n'a pas forcément sa place ici, mais le préciser est pas dénué de sens puisque ça appuie la position de la personne qui a fait la remarque contre une éventuelle classe divine.)
En passant, ton message n'a pas grand chose à voir avec la suggestion non plus.

**Reply by aureochocob0n** [reply]
Je comprends toujours pas le sens de ton commentaire 🤣, si y en a deux comme tu les cites indirectement qui se comportent "mal" selon toi t'as qu'à les ignorer. De 1, c'est un rp donc si t'es contre ce genre rôles tu n'a toujours pas compris le sens du jeu auquel tu joues depuis tant de temps. De 2, c'est pas à cause de 2 personnes que l'on va empêcher une classe d'être crée

**Reply by Jajadu69** [reply]
Ahhhh cyrpa j'ai compris.  Tu disais que la remarque de carsebel sur le fait que certains allaient choper le melon avait sa place ici. Puisque ça appuie la position de carsebel sur la proposition d'une classe divine, soit non.  C'est ça ?

**Reply by Cyrpa** [reply]
Ben apprends à lire, que veux-tu que je te dise ? 
Je visais qu'une personne qui plus est, doucement les interprétations foireuses merci.
Après, y a une différence entre jouer un rôle et faire preuve d'une mauvaise foi pure et dure, constatée chez ce joueur par plusieurs.
Ensuite, c'était qu'un argument (comme je l'ai expliqué dans le précédent message d'ailleurs, vraiment si tu l'as pas compris n'y réponds pas stp on gagnerait tous du temps) comme un autre, il a pas forcément pour objectif d'empêcher la classe d'être créée, mais juste de faire réfléchir quant à ladite classe.
Pour finir, j'aimerais préciser que je n'ai en aucun cas précisé que c'était mon avis, je ne faisais que répondre à jaja pour lui expliquer le rapport avec la suggestion.
En espérant que tu ne comprennes pas cette réponse non plus mais que tu fasses preuve de sagesse en n'y répondant pas, 
Bonne journée. 👍

**Reply by Cyrpa** [reply]
Yep merci jaja de clarifier mon propos pour ton ami

**Reply by Jajadu69** [reply]
Je tiens à dire que je ne l'avais pas compris non plus . D'ailleurs choco m'avait demandé mon avis donc c'est vraiment pas lui qui ne sait pas lire c'est moi. 
De plus tu dis maintenant que tu ne visais qu'une seule personne alors que ta réponse avant était au pluriel. Fin ça laisse la place à la mauvaise interprétation.

**Reply by Cyrpa** [reply]
C'était pour ne viser personne précisément, car c'est pas très sympa, mais puisque chocomachin l'a interprété je suis bien obligé de lui dire. 
Bref si c'est bon on peut laisser place aux messages utiles à la suggestion merci.

**Reply by Ravenclaw-l** [reply]
Je trouve l'idée de créer un T6 plutôt intéressante. Il n'y a plus de proposition d'évolution à partir du palier 5, et je trouve ça dommage que des niveaux 115, 120 ou 125 se retrouvent avec la même classe qu'un niveau 80. À titre d'exemple, c'est comme si un niveau 80 se retrouvait avec la même classe qu'un niveau 35, 40 ou 45. Entre le niveau 8 et le niveau 80, on retrouve 5 paliers différents. Après ça, plus rien, alors que le conseil te rappelle souvent que "le jeu ne s'arrête pas au niveau 100" ^^. 
Si l'on prend en compte le nombre d'xp nécessaire pour passer de niveaux à partir du niveau 100, on se rend compte que l'on garde les mêmes classes beaucoup trop longtemps. Ad vitam aeternam en fait puisqu'il y en a pas d'autres. 
Donc, il faudrait soit créer un T6, soit décaler le passage des classes et pouvoir accéder au T5 qu'à partir du niveau 100, par exemple.

**Reply by LaMangousteRose** [reply]
C pas que des lvl 100 qui sont pour le T6?
Après je dis ça je dis rien c'est un idée mais toutes les autres personnes qui sont pas lvl 100 vont avoir le seum si ça se passe.

**Reply by LaMangousteRose** [reply]
Ou sinon la classe on la mets qu'en T1 mdr

**Reply by StrangerIM** [reply]
> C pas que des lvl 100 qui sont pour le T6?
> 
> Après je dis ça je dis rien c'est un idée mais toutes les autres personnes qui sont pas lvl 100 vont avoir le seum si ça se passe.

Pourquoi ça ?

**Reply by LaMangousteRose** [reply]
Car ils ont déjà moins de choix de classe que les lvl 80 et + et il y en a qui ne savent pas jouer aucune des classes

**Reply by LaMangousteRose** [reply]
Ils seront jaloux

**Reply by Ravenclaw-l** [reply]
Je ne pense pas que la jalousie soit une raison valable pour arrêter de développer le jeu.

**Reply by LaMangousteRose** [reply]
D'accord d'accord mais il n'y a pas tellement de lvl 100 et + donc ça ne changera pas grand chose pour la commu

**Comment by Carsebel** [comment]
> T'es premier degré là ?
Oui tout à fait

**Comment by OrionYp** [comment]
Je suis d'accord que ça pourrait être cool d'augmenter l'importance du lore, mais une nouvelle classe bof...
Et sinon je suis totalement contre un sixième palier de classes.

**Reply by Cirpacha** [reply]
cool mais t'as oublié d'argumenter

**Reply by Gabrielre14** [reply]
Moi aussi. 

Je vois plutôt le moine comme une nouvelle classe du palier T5, comme le mage. L'idée de la classe en rapport avec le kyutisme est excellente, mais je pense qu'il faudrait mieux s'arrêter là. Ajouter une classe, un peu comme l'opposé du mage (le moine / prêtre kyutisme est un fervent religieux là où le mage explore une puissance contre-nature) est suffisant. Il pourrait d'ailleurs être un sorte de counter. Ou bien ils se contreraient mutuellement, ce qui pourrait être sympa pour incrémenter dans l'équilibrage le lore de ces classes. 

La référence est sympa mais il est inutile de trop la développer. Le Kyutisme reste à la base une sorte de blague^^

**Reply by Carsebel** [reply]
Heureusement qu'il n'y a pas que ta voix qui compte.

**Reply by OrionYp** [reply]
Il y a déjà beaucoup de classes t5 si on veut toutes les essayer ça prend 6 mois et l'équilibrage entre elles est bancal, donc si on rajoute une classe ce sera encore pire.
Pour un sixième palier de classe je ne pense pas que c'est une bonne idée car il y a déjà une énorme différence entre t4 et t5 (un combat entre t4 et t5 l'issue est assez prévisible..) pas spécialement envie de subir ça au niveau 100 comme certains ici le proposent.

Heureusement qu'il n'y a pas que ta voix qui compte non plus.

**Reply by Cirpacha** [reply]
c'est le principe d'un nouveau palier, ça permettrait aux lvl 100+ de pas perdre contre des lvl 80 qui ont juste eu la chance de drop de bons items (valorisation de l'expérience des joueurs). Et en passant ça nous changerait un peu de nos classes habituelles...

**Reply by OrionYp** [reply]
Alors un vrai nouveau palier, avec de nouvelles attaques et des noms différents

**Reply by kirit000** [reply]
Mdrr gab je pense que le mage souffre déjà assez contre le paladin, pour une nouvelle classe ok mais on ne sais pas si ça sera mieux équilibrée ou si ça va empirer. Il ne joue pas qu'un rôle en fight, n'oubliez pas qu'il a sa place dans l'aventure et dans l'île. En vrai la classe a du potentiel sur les rushs en revenant sur ta proposition de l'altération de pv gab, s'il est fort en fight et pour l'île, en aventure on ira lui mettre des PV similaires au canonnier plus ou moins.

**Reply by Gabrielre14** [reply]
Concernant l'altération "soin", avec des telles stats ce ne serait pas une attaque à 0 de souffle comme le repos mais bien un truc quasiment aussi cher que l'attaque maudite x)
Après tout ceci n'est que de l'équilibrage, mais je trouve intéressant d'avoir une rivalité de classes en fight comme dans le lore (un peu comme la rivalité canonnier / paladin en fight qui pourrait être cool d'ajouter à la description des classes).

**Comment by GabTheminor** [comment]
J’adore l’idée, je suis totalement pour. Les gens vont prendre que ça 😂

**Comment by Electek** [comment]
Je relance le post vue que ça m'a l'air d'être plutôt une bonne idée :3

**Reply by BastLast** [reply]
on peut pas simplement rajouter une nouvelle classe dans le jeu alors va falloir etre très très patient, c'est pas un non mais c'est pas pour bientot l'ajout de nouvelle classe

**Reply by Electek** [reply]
Franchement je voulais juste savoir si ct un oui ou un non 😂. Mais oe je comprends que c'est bcp de taff et que c'est pas la prio. En plus de ça devoir équilibré encore une autre classe ça va être relou. Mais je suis content que ça pourrait potentiellement être ajouté même si c'est dans un futur lointain ^^
